torch/_inductor/config.py

```diff
@@ -141,64 +149,67 @@ def decide_compile_threads():
         )
 
 
-compile_threads = decide_compile_threads()
+def is_fbcode():
```

Comment:
There is also a version of this in ConfigMixin.  Which one gets called?  Could lead to confusing bugs if one edits the wrong one.